lib/animina/accounts/resources/basic_user.ex

```diff
@@ -29,9 +29,21 @@ defmodule Animina.Accounts.BasicUser do
     end
 
     attribute :birthday, :date, allow_nil?: false
-    attribute :zip_code, :string, allow_nil?: false
+
+    attribute :zip_code, :string do
+      constraints max_length: 5,
```

Comment:
I am not a big fan of the validation error here. "Postal code length must be greater than or equal to 5" is not correct. It should be something like "Postal code length must be 5".

Either we have to "fix" the validation error or we have to find a solution to use some sort of `length: 5`

Reply:
I've added a custom validation for the zip_code attribute, since Ash.Type.String constraints does not support a length validation.